src/status_im/ui/screens/activity_center/views.cljs

```diff
@@ -219,19 +219,20 @@
 
 (defn activity-center
   []
-  (reagent/create-class
-   {:component-did-mount #(>evt [:activity-center.notifications/fetch-first-page])
-    :reagent-render
-    (fn []
-      (let [notifications (<sub [:activity-center/filtered-notifications])
-            window-width  (<sub [:dimensions/window-width])]
-        [safe-area/view {:style {:flex 1}}
-         [rn/view {:style {:width window-width
-                           :flex  1}}
-          [header]
-          [rn/flat-list {:content-container-style {:flex-grow 1}
-                         :data                    notifications
-                         :empty-component         [empty-tab]
-                         :key-fn                  :id
-                         :on-end-reached          #(>evt [:activity-center.notifications/fetch-next-page])
-                         :render-fn               render-notification}]]]))}))
+  [:f>
+   (fn []
+     (let [notifications         (<sub [:activity-center/filtered-notifications])
```

Comment:
[Code style] Let bindings have extra space.

Reply:
removed it